source/textInfos/__init__.py

```diff
@@ -287,6 +288,9 @@ def _logBadSequenceTypes(sequence: SpeechSequence, shouldRaise: bool = True):
 	return speech.types.logBadSequenceTypes(sequence, raiseExceptionOnError=shouldRaise)
 
 
+TextInfoObjT = Union["NVDAObjects.NVDAObject", "treeInterceptorHandler.TreeInterceptor"]

```

Comment:
An alternative to adding this new type is to instead use `documentBase.TextContainerObject`.
Both NVDAObject and TreeInterceptor inherit from this.

Reply:
Thanks, this also appears to be more correct. `cursorManager.CursorManager` should also be included, another case which inherits from TextContainerObject